swagger-backend.yaml

```diff
@@ -63,6 +63,21 @@ definitions:
         description: >-
           Exposes the error message generated at the backend when there is an
           error (example HTTP code 500).
+  playbook-executions-response:
+    type: array
```

Comment:
Something is not right here: the generated model looks weird (especially `models.PlaybookExecutionsResponseItems0`
Maybe it should be something like:
```
playbook-executions-response:
  schema:
    x-go-type:
       type: PlaybookExecutionsResponse
       import:
         package: github.com/project-flotta/flotta-operator/models
         alias: commonmodel
```
or even `x-go-type` should be under `schema`@https://github.com/project-flotta/flotta-operator/pull/263/files#diff-8e6b4f2eed3a98c3e53c8c64be9501bf449db857bf32e695f9a6c5ec5b490a9fR277 and separate response definition should be removed?

Reply:
Done. Can you please double check it?

Reply:
LGTM